app/utils/request_session.py

```diff
@@ -0,0 +1,8 @@
+import aiohttp
+
+
+session = aiohttp.ClientSession()
```

Comment:
A new client will be created every time this file is imported. In this case, it gets initialized 4 different times. It's not a singleton.

Reply:
i love python